packages/e2e-tests/specs/blocks/preformatted.test.js

```diff
@@ -22,11 +22,13 @@ describe( 'Preformatted', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 
 		await page.keyboard.press( 'Escape' );
+		await page.waitForSelector( 'button[aria-label="More options"]' );
```

Comment:
Why does this need to exist. It's a code smell that will continue to be an issue in other tests.